source/Octopus.Cli/Commands/Releases/PromoteReleaseCommand.cs

```diff
@@ -25,8 +25,10 @@ public PromoteReleaseCommand(IOctopusAsyncRepositoryFactory repositoryFactory, I
             options.Add<string>("from=", "Name or ID of the environment to get the current deployment from, e.g., 'Staging' or 'Environments-2'.", v => FromEnvironmentNameOrId = v);
             options.Add<string>("to=|deployTo=", "Name or ID of the environment to deploy to, e.g., 'Production' or 'Environments-1'.", v => DeployToEnvironmentNamesOrIds.Add(v));
             options.Add<bool>("updateVariables", "Overwrite the variable snapshot for the release by re-importing the variables from the project", v => UpdateVariableSnapshot = true);
+            options.Add<bool>("latestSuccessful", "Use the latest successful release to promote", v => UseLatestSuccessfulRelease = true);
```

Comment:
I think this is going to change the behaviour if it has a default value of true.

Reply:
from my understanding, the default will always be false, unless the use sets this option in their call - 
```
promote-release --project=projectA --from=Dev --to=Test --latestSuccessful ...
```

Just tested this, and if you ommit `--latestSuccessful` then the values comes out as false.

Reply:
You're right sorry, I misread.